pkgs/development/python-modules/dukpy/default.nix

```diff
@@ -21,6 +21,12 @@ buildPythonPackage rec {
     hash = "sha256-8RDMz9SfBoUe7LQ9/atsZlJ/2uwLUb0hZxeYdsUOGpU=";
   };
 
+  postPatch = ''
+    substituteInPlace tests/test_webassets_filter.py \
+      --replace-fail "class PyTestTemp" "class _Temp" \
+      --replace-fail "PyTestTemp" "Temp"
+  '';
```

Comment:
Do you know what’s going on here? Was it broken already or do the changes in this PR somehow make this necessary?

Reply:
Ah, I see, it’s adapting for the changes in `webassets.test`. Okay, that seems fine then; hopefully upstream will release and the ecosystem will adapt at some point so that we no longer need to carry this.